test/mainnet/invariants.spec.sol

```diff
@@ -84,19 +83,19 @@ contract VaultMainnetInvariantsTest is Test, MainnetActors {
 
         // Test the convertToAssets function
         uint256 convertedAssets = vault.convertToAssets(shares);
-        assertApproxEqRel(convertedAssets, assets, 1e14, "Converted assets should equal the original assets");
+        assertApproxEqAbs(convertedAssets, assets, 3, "Converted assets should equal the original assets");
```

Comment:
relative error used was very high because at low wei amounts eg. 1000, 2 wei of error is 0.2%. 

but that error does not scale with size of input.